test/jest/authService.test.js

```diff
@@ -269,6 +269,21 @@ describe('AuthService', () => {
       return res;
     });
 
+    test('should end up with isPending === false', async () => {
+      expect.assertions(1);
```

Comment:
Is this required when you have an async callback?   I thought this was only required when it WASN'T async

Reply:
it would be helpful for the async test since the callback might be missed.

by a quick search:
> If you expect a promise to be rejected, use the .catch method. Make sure to add expect.assertions to verify that a certain number of assertions are called. Otherwise, a fulfilled promise would not fail the test.

From: https://jestjs.io/docs/en/asynchronous#promises